OneSignalSDK/onesignal/src/main/java/com/onesignal/GenerateNotification.java

```diff
@@ -929,13 +910,6 @@ private static int getDrawableId(String name) {
       return contextResources.getIdentifier(name, "drawable", packageName);
    }
 
-   private static boolean isSoundEnabled(JSONObject fcmJson) {
```

Comment:
Keep this method, just remove the `OneSignal.getSoundEnabled();` check. Same reason noted above about the Android Notification Categories